src/components/molecules/Header.tsx

```diff
@@ -7,7 +7,7 @@ interface Props {
 
 const Header = ({ userName, online }: Props) => {
   return (
-    <header className="flex justify-end h-24 w-full bg-gray px-8 py-4">
+    <header className="text-white flex justify-end h-24 w-full bg-gray px-8 py-4">
```

Comment:
The error persists on the footer

![image](https://user-images.githubusercontent.com/76726139/206778837-8a11785d-20bf-406e-b94e-14350804014b.png)

probably we should change body default color instead of just adding text-white to all our p tags